packages/web/components/CardResultsProvider.tsx

```diff
@@ -37,26 +62,50 @@ export default function CardResultsProvider({
   serverCards: ICard[];
 }) {
   serverCards.sort(compareCards);
-  const [cards, setCards] = useState(serverCards);
+  const [cards, setCards] = useState(serverCards.slice(0, 5));
   const [indexedCards, setIndexedCards] = useState(
     getIndexedCards(serverCards)
   );
+  const [hasMoreCards, setHasMoreCards] = useState(true);
+
+  const fetchMoreCards = async () => {
+    try {
+      const lastCard = cards[cards.length - 1];
+      const newCards = await fetchCardsFromSupabase(lastCard?.id);
+      console.log("Fetched new cards:", newCards);
```

Comment:
Intentionally left?

Reply:
Don't follow. Same with below.

Reply:
Did you mean to leave the console.log statement? Usually good practice to remove non error logs before prod.